src/web/components/SharingPermission/TypeFilter.tsx

```diff
@@ -1,35 +1,35 @@
 import clsx from 'clsx';
 
-import { ParticipantTypeDTO } from '../../../api/entities/ParticipantType';
+import { ClientType } from '../../../api/services/adminServiceHelpers';
 
 import './TypeFilter.scss';
 
 type TypeButtonProps = {
-  type: ParticipantTypeDTO;
+  type: ClientType;
   isSelected: boolean;
-  onTypeSelect: (selectedTypeId: number) => void;
+  onTypeSelect: (selectedTypeId: ClientType) => void;
 };
 
 export function TypeButton({ type, isSelected, onTypeSelect }: TypeButtonProps) {
   return (
     <button
       type='button'
       className={clsx('type-button', { selected: isSelected })}
-      onClick={() => onTypeSelect(type.id)}
+      onClick={() => onTypeSelect(type)}
     >
-      {type.typeName}
+      {type}
```

Comment:
I believe this needs the same treatment for displaying the friendly desciption:
```
{ClientTypeDescriptions[type]}
```
![Uploading image.png…]()

Reply:
Done!